setup.py

```diff
@@ -30,7 +30,7 @@ def long_description():
     with open(os.path.join(BASE_PATH, 'README.rst')) as readme:
         result = readme.read()
     result += '\n\n'
-    with open(os.path.join(BASE_PATH, 'CHANGES.rst')) as changes:
+    with open(os.path.join(BASE_PATH, 'CHANGES.md')) as changes:
```

Comment:
THANK YOU! But let's exclude it from here altogether -- mixing .rst and .md would not lead to anything good there ;)   We could just add a pointer to README.rst to point to it...

alternative is to e.g. use pypandoc at setup time and convert, like apparently we did in datalad : https://github.com/datalad/datalad/blob/HEAD/_datalad_build_support/setup.py#L299   but I do not think it is worth it.

Reply:
I think I might just change all our text files to .md. I think md is more commonly used now, and then they would be consistent

Reply:
ok. iirc there might be need for some metadata field in `setup()` to signal that it is markdown... don't remember

meanwhile I will see if I can cut release of 0.7.0 on top of which we would work out automation

Reply:
@tmorrell while we are on a stretch -- will you have time to convert others to .md too?

Reply:
All done! I had to update coveralls to run from this repo, and also swapped out the token.

Reply:
I don't have a good way to test coveralls before merging....so it might need some adjustment if it doesn't start working.